api_docs/construct-narrow.md

```diff
@@ -45,6 +45,13 @@ operator, search the current user's personal message history. See
 [searching shared history](/help/search-for-messages#searching-shared-history)
 for details.
 
+**Changes**: In Zulip 7.0 (feature level 177), support was added for
+  three narrows related to direct messages: `is:dm`, `dm` and
+  `dm-including`. The `dm` operator replaced and deprecated the
+  `pm-with` operator. The `is:dm` narrow replaced and deprecated the
+  `is:private` narrow. The `dm-including` operator replaced and
+  deprecated the `group-pm-with` operator.
```

Comment:
I think this is the place where I'd spell out that `group-pm-with` had slightly different semantics from `dm-including` -- since this is the detailed documentation page one should use when one is interested in precise search operator semantics.

Reply:
Yes, this makes sense. Expanded this comment in the updated version.